server/api/utils/clients/async_nuclio.py

```diff
@@ -38,7 +38,8 @@
 class Client:
     def __init__(self, auth_info: mlrun.common.schemas.AuthInfo):
         self._session = None
-        self._auth = aiohttp.BasicAuth(auth_info.username, auth_info.session)
+        login = auth_info.username
+        self._auth = aiohttp.BasicAuth(login, auth_info.session) if login else None
```

Comment:
Why not checking for `auth_info` to be None? 
```suggestion
        self._auth = aiohttp.BasicAuth(lauth_info.username, auth_info.session) if auth_info else None
```

Reply:
@rokatyy, the issue is that `auth_info.username` is `None`, not that `auth_info` is `None`.

Reply:
`auth_info` can't be `None` because the server route always creates this object afaict.